routes/api/bookin.js

```diff
@@ -5,20 +5,19 @@ const passport = require("passport");
 const Hotel = require("../../models/Hotel");
 const Booking = require("../../models/booking");
 const Customer = require("../../models/customer");
-const City = require("../../models/city")
+const City = require("../../models/city");
 const User = require("../../models/User");
 
-const confirmEmail = require('../../email/confirmationEmail')
-const welcomeEmail = require('../../email/welcomeEmail')
+const confirmEmail = require("../../email/confirmationEmail");
```

Comment:
1000 lines changed... don't feel like going through all this... what did you change?

Reply:
I made changes only for **router.post("/guest-history", (req, res) => { ...} ** . Please look down below for my comments